Description: Disabled assembler code.
 Some fixes committed upstream since 2004. Next version (1.5.1) fixes new issues
 so keep disabled for now.
Forwarded: no
Bug-Debian: http://bugs.debian.org/267986
Last-Update: 2010-12-26
--- a/source/SoundTouch/cpu_detect_x86_gcc.cpp
+++ b/source/SoundTouch/cpu_detect_x86_gcc.cpp
@@ -68,7 +68,7 @@
 /// Checks which instruction set extensions are supported by the CPU.
 uint detectCPUextensions(void)
 {
-#if (!(ALLOW_X86_OPTIMIZATIONS) || !(__GNUC__))
+#if 1 // (!(ALLOW_X86_OPTIMIZATIONS) || !(__GNUC__))
 
     return 0; // always disable extensions on non-x86 platforms.
 
